src/WebJobs.Extensions.OpenAI/Assistants/AssistantService.cs

```diff
@@ -56,20 +61,52 @@ record InternalChatState(string Id, AssistantStateEntity Metadata, List<ChatMess
 
         this.logger = loggerFactory.CreateLogger<DefaultAssistantService>();
 
-        string connectionStringName = openAiConfigOptions.Value.StorageConnectionName;
+        IConfigurationSection storageConfig = configuration.GetSection(this.chatStorage);
+        this.logger.LogInformation("This is the storageConfig: {storageConfig} ", storageConfig);
 
-        // Set connection string name to be AzureWebJobsStorage if it's null or empty
-        if (string.IsNullOrEmpty(connectionStringName))
+        if (!string.IsNullOrEmpty(storageConfig.Value))
         {
-            connectionStringName = "AzureWebJobsStorage";
+            this.logger.LogInformation("This is the storageConfig.value: {storageConfig} ", storageConfig.Value);
         }
 
-        this.logger.LogInformation("Using {ConnectionStringName} for table storage connection string name", connectionStringName);
+        string storageAccountUri = storageConfig["tableServiceUri"];
 
-        string connectionString = configuration.GetValue<string>(connectionStringName);
+        this.logger.LogInformation("This is the storageAccountUri: {storageAccountUri} ", storageAccountUri);
 
-        this.tableServiceClient = new TableServiceClient(connectionString);
+        
+        // Check if URI for table storage is present
+        if (!string.IsNullOrEmpty(storageAccountUri))
```

Comment:
Should we also be looking for other configuration properties, like `accountName`, which Functions does for managed identity with AzureWebJobsStorage?

Reply:
Confirmed with Eamon to leave it out for this case, since it is very specific AzureWebJobsStorage.